ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -17,24 +17,9 @@
 except ImportError as e:
     pymqiException = e
     pymqi = None
-
-log = logging.getLogger(__file__)
-
-
-class IbmMqCheck(AgentCheck):
-
-    METRIC_PREFIX = 'ibm_mq'
-
-    SERVICE_CHECK = 'ibm_mq.can_connect'
-
-    QUEUE_MANAGER_SERVICE_CHECK = 'ibm_mq.queue_manager'
-    QUEUE_SERVICE_CHECK = 'ibm_mq.queue'
-
-    CHANNEL_SERVICE_CHECK = 'ibm_mq.channel'
-    CHANNEL_STATUS_SERVICE_CHECK = 'ibm_mq.channel.status'
-
-    CHANNEL_COUNT_CHECK = 'ibm_mq.channel.count'
-
+else:
+    # pymqi is not be available on win/macOS when running e2e
```

Comment:
pymqi is avaiblable on mac, but you need to install it: https://github.com/DataDog/integrations-core/blob/48cb9081d5b992076955afba2d13a5276c61ee63/ibm_mq/tests/README.md

Reply:
I update the working to be more explicit about that.